### ISSUE TITLE
Document JS Engine requirement on Getting Started guide

### ISSUE THREAD

**Issue by Zabeus** [Shopify/dashing#624]
Please update the Getting Started guide at http://dashing.io/#setup to mention that you need a Javascript runtime such as NodeJS installed before running "dashing start".
(it may be obvious to Ruby web developers, but a lot of people like me just want to try dashing and have very little experience with Ruby)

Related issue: #279 .

**Comment by tylermauthe** [comment]
Yeah, sorry about that. We've discussed adding a gem that would enable portability without needing to install a separate JS engine.

@pushmatrix - This is something else for the todo list, adding therubyracer gem for JavaScript portability on Linux.

**Comment by pushmatrix** [comment]
:+1:
